examples/java/src/test/java/org/apache/beam/examples/WordCountIT.java

```diff
@@ -68,15 +66,14 @@
   public void testE2EWordCount() throws Exception {
     PipelineOptionsFactory.register(WordCountITOptions.class);
     WordCountITOptions options = TestPipeline.testingPipelineOptions().as(WordCountITOptions.class);
+
+    IOChannelUtils.registerStandardIOFactories(options);
```

Comment:
Currently, this method only support GCS. Probably we need to support more in order to make WordCountIT run on other platform/runner.